pilot/cmd/pilot-agent/status/server.go

```diff
@@ -414,16 +426,8 @@ func (s *Server) handleAppProbe(w http.ResponseWriter, req *http.Request) {
 		_, _ = w.Write([]byte(fmt.Sprintf("app prober config does not exists for %v", path)))
 		return
 	}
-
-	// Construct a request sent to the application.
-	httpClient := &http.Client{
-		Timeout: time.Duration(prober.TimeoutSeconds) * time.Second,
-		// We skip the verification since kubelet skips the verification for HTTPS prober as well
-		// https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
-		Transport: &http.Transport{
-			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-		},
-	}
+	// get the http client must exist because
```

Comment:
Update the comment? What if the connection initiated by a cached client is broken, does the client automatically reconnect?

Reply:
@hzxuzhonghu could you please update the PR?

Reply:
@JimmyCYJ Yes, httpClient will automatically establish a connection if no connection available when send a request

Reply:
@JimmyCYJ Done